src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1391,5 +1391,9 @@ internal enum ErrorCode
         ERR_TupleExplicitNamesOnAllMembersOrNone = 8204,
 
         ERR_PredefinedTypeMemberNotFoundInAssembly = 8205,
+        ERR_MissingDeconstruct = 8206,
```

Comment:
I filed issue #11405. 
From discussion with Vlad, the idea was to do that re-organization when we get close to code complete.